ui/page/privacy/account_mixer_page.go

```diff
@@ -68,175 +68,176 @@ func (pg *AccountMixerPage) OnNavigatedTo() {
-									}
-								}
-
-								return components.MixerInfoContentWrapper(gtx, pg.Load, func(gtx C) D {
-									return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-										layout.Rigid(func(gtx C) D {
-											return layout.Flex{Spacing: layout.SpaceBetween, Alignment: layout.Middle}.Layout(gtx,
-												layout.Rigid(func(gtx C) D {
-													txt := pg.Theme.Label(values.TextSize14, "Unmixed balance")
-													txt.Color = pg.Theme.Color.GrayText2
-													return txt.Layout(gtx)
-												}),
-												layout.Rigid(func(gtx C) D {
-													return components.LayoutBalance(gtx, pg.Load, unmixedBalance)
-												}),
-											)
-										}),
-										layout.Rigid(func(gtx C) D {
-											return layout.Center.Layout(gtx, pg.Theme.Icons.ArrowDownIcon.Layout24dp)
-										}),
-										layout.Rigid(func(gtx C) D {
-											return layout.Flex{Spacing: layout.SpaceBetween, Alignment: layout.Middle}.Layout(gtx,
-												layout.Rigid(func(gtx C) D {
-													t := pg.Theme.Label(values.TextSize14, "Mixed balance")
-													t.Color = pg.Theme.Color.GrayText2
-													return t.Layout(gtx)
-												}),
+func (pg *AccountMixerPage) MixerInfoLayout(gtx C, l *load.Load, mixerActive bool, button layout.Widget) D {
+	mixedBalance := "0.00"
+	unmixedBalance := "0.00"
+	accounts, _ := pg.wallet.GetAccountsRaw()
+
+	for _, acct := range accounts.Acc {
+		if acct.Number == pg.wallet.MixedAccountNumber() {
+			mixedBalance = dcrutil.Amount(acct.TotalBalance).String()
+		} else if acct.Number == pg.wallet.UnmixedAccountNumber() {
+			unmixedBalance = dcrutil.Amount(acct.TotalBalance).String()
+		}
+	}
+	return l.Theme.Card().Layout(gtx, func(gtx C) D {
+		txt := l.Theme.H6(values.String(values.StrBalance))
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.UniformInset(values.MarginPadding40).Layout(gtx, func(gtx C) D {
+					return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							return layout.Inset{Bottom: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
+								return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
+									layout.Rigid(txt.Layout),
+									layout.Rigid(func(gtx C) D {
+										return layout.Inset{Left: values.MarginPadding5}.Layout(gtx, func(gtx C) D {
+											return components.LayoutBalance(gtx, pg.Load, pg.totalBalance.String())
+										})
+									}),
+									layout.Flexed(1, func(gtx C) D {
+										return layout.E.Layout(gtx, func(gtx C) D {
+											txt := l.Theme.H6(values.String(values.StrMix))
+											return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
 												layout.Rigid(func(gtx C) D {
-													return components.LayoutBalance(gtx, pg.Load, mixedBalance)
+													return layout.Inset{Right: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
```

Comment:
Should be : `return layout.Inset{Right: values.MarginPadding10}.Layout(gtx, txt.Layout)`